docs/UPGRADING.md

```diff
@@ -0,0 +1,91 @@
+# Angular Package Format
+
+The npm package for the IDS Enterprise Angular components (`ids-enterprise-ng`) has been converted into a compiled package (using the [APF][#APF])
+
+For information on how to create these libraries, see [Creating A Library][#CAL].
+
+## Component Library Consumers
+
+Consuming the `ids-enterprise-ng` package will require changes to any projects referencing it.
+
+These instructions assume you will be running the latest versions of `@angular/cli` and `@angular/core`. It is recommended that you review the information on <http://update.anguar.com> before updating.
+
+These are the steps for upgrading existing projects:
+
+```sh
+npm install @angular/cli -g
+npm install @angular/cli
+ng update @angular/cli
+ng update @angular/core
+```
+
+You will need to fix any issues raised, as these will depend on the dependency tree created by the packages you use and what version you are upgrading from.
+
+Install the latest `ids-enterprise` and `ids-enterprise-ng` components.
```

Comment:
Since we are moving `ids-enterprise` to a dependency, they won't have to manually install it, like we talked about in chat.